tests/components/downloader/test_config_flow.py

```diff
@@ -32,10 +31,7 @@ async def test_user_form(hass: HomeAssistant) -> None:
         )
         assert result["type"] is FlowResultType.FORM
 
-        with patch(
-            "homeassistant.components.downloader.config_flow.DownloaderConfigFlow._validate_input",
-            side_effect=DirectoryDoesNotExist,
-        ):
+        with patch("os.path.isdir", return_value=False):
             assert result["type"] is FlowResultType.FORM
             assert result["step_id"] == "user"
             assert result["errors"] == {"base": "cannot_connect"}
```

Comment:
Side note: `cannot_connect` as error string doesn't seem appropriate for a missing directory. I'd use another string that better matches the error we're dealing with.

Reply:
You want that in this PR scope, or a new one?

Reply:
Either here or in a different PR.